src/ovirt-engine-hyper-upgrade

```diff
@@ -203,10 +191,13 @@ class UpgradeHelper(object):
         Returns: Empty list or yum update output
         '''
 
-        print("Consulting ovirt-engine-setup and ovirt-engine-dwh"
-              "packages in the system for update...")
+        yum_update_engine_cmd = list(self.YUM_UPDATE_CMD)
+        yum_update_engine_cmd.extend(
+            'ovirt-engine-setup*',
+            'ovirt-engine-dwh-setup*'
```

Comment:
I think we used to suggest:

 yum update ovirt-engine\*setup\*

At the time, this included also ovirt-engine-reports. Perhaps today these are identical.

Reply:
During my tests if I don't update ovirt-engine-dwh-setup* the engine-setup fails requiring an updated ovirt-engine-dwh-setup package. @didib